src/content/docs/browser/new-relic-browser/configuration/obfuscate-browser-agent-data.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: Obfuscate Browser Agent Data
+tags:
+  - Browser monitoring
+  - Obfuscation
+metaDescription: "The agent can be configured to obfuscate the data it sends."
+---
+
+<Callout variant="important">
+This feature is currently available for those using the copy/paste or NPM browser installation methods. There is currently no UI or NerdGraph configuration options available. We are continuing to work on improving access to these and other configuration options.
+</Callout>
+
+While New Relic's recommendation is to abstain from utilizing sensitive information in the public structure of your application, we also understand that it is not always feasible to remove all traces of said information. The browser agent can be configured to selectively obfuscate data in every payload it sends.  This can be useful if your application uses sentitive data in places that the agent captures, such as navigation paths, error messages, and more.
+
+## How it works [#how-it-works]
+
+As of browser agent [version 1216](/docs/release-notes/new-relic-browser-release-notes/browser-agent-release-notes) and higher, obfuscation rules can be applied to outgoing harvest payloads. 
+
+To set up these rules, you'll need to configure the following browser agent properties: 
+* `init.obfuscate` contains an array of selectors and replacements which will be utilized to modify each harvest before sending.
```

Comment:
If this is just one property with two tips, could we revise like this (and make them parallel)?

* The property `init.obfuscate` contains an array of selectors and replacements which will be used to modify each harvest before sending.
* Since obfuscation currently requires you to use the copy/paste installation method, you’ll need to manually edit your JavaScript snippet and set the `obfuscate` array to contain your obfuscation conditions.

Reply:
you can use this with NPM as well

Reply:
updated to match suggestion with slight modification about NPM